Dockerfile

```diff
@@ -34,7 +34,8 @@ RUN apt-get update && \
         swig \
         xmlto \
         zlib1g-dev && \
-    apt-get clean && \
+        gcc-multilib 
```

Comment:
Are you not running the supplied 32-bit Docker instance or something like that? This should not be necessary, and causes problems for certain packages.